arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/components/ResourceInstanceMultiSelectWidgetEditor.vue

```diff
@@ -53,14 +53,15 @@ watch(
 
 const resourceResultsCurrentCount = computed(() => options.value.length);
 
-async function getOptions(page: number) {
+async function getOptions(page: number, searchTerm: string) {
     try {
         isLoading.value = true;
 
         const resourceData = await fetchRelatableResources(
             props.graphSlug,
             props.nodeAlias,
             page,
+            searchTerm || "",
```

Comment:
IMO this should not always evaluate to a string. it could be handled for at the `api.ts` level instead.